2015/day2/README.md

```diff
@@ -1,3 +1,3 @@
-# Advent of Code 2015 Day 1
+# Advent of Code 2015 Day 2 <!-- Stupid! -->
```

Comment:
I'm not mergin this pr if it have this comment.

Reply:
Solved